instaclustr/resource_cluster.go

```diff
@@ -592,6 +592,45 @@ func resourceCluster() *schema.Resource {
 					return
 				},
 			},
+			"kafka_connect_credential": {
+				Type:	  schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem:	  &schema.Resource {
+					Schema: map[string]*schema.Schema {
+						"aws_access_key": {
+							Type:     schema.TypeString,
```

Comment:
No GCP?

Reply:
previously I was trying to make it in time for the prod freeze, so it's just what's existing. I'll add the GCP support for KC along with this PR too

Reply:
Actually, maybe it's better to be done in another ticket

Reply:
https://instaclustr.atlassian.net/browse/INS-17136